test/runtests.jl

```diff
@@ -83,11 +91,15 @@ end
         ]
         ry = AbstractFFTs.rfft(x, dims)
         @test ry ≈ fftw_rfft
-        P = plan_rfft(x, dims)
-        @test eltype(P) === Int
-        @test P * x ≈ fftw_rfft
-        @test P \ (P * x) ≈ x
-        @test fftdims(P) == dims
+        # test plan_rfft and also inv and plan_inv of plan_irfft, which should all give 
+        # functionally identical plans
+        for P in [plan_rfft(x, dims), inv(plan_irfft(ry, size(x, dims), dims)), 
+                  AbstractFFTs.plan_inv(plan_irfft(ry, size(x, dims), dims))]
+            @test eltype(P) <: Real
```

Comment:
Other than add the loop over `P`, the only other change I made was to this assertion. Previously the assertion was that the `eltype` should be `Int` here. That doesn't seem right (for `FFTW` the `eltype` would be `Float64`); rather than thinking too hard about what the test plan implementation should be doing (it's probably underspecified), I just made the check a little looser.